src/renderer/screens/config/ConfigScreens/WorkspaceScreen.js

```diff
@@ -120,9 +122,43 @@ class WorkspaceScreen extends Component {
           <h4>TRUFFLE PROJECTS</h4>
           <div className="Row">
             <div className="RowItem">
+              {this.props.config.validationErrors["workspace.project"] && (
+                <div>
+                  <div className="ValidationError">
+                    {
+                      this.props.config.validationErrors["workspace.project"]
+                        .message
+                    }
+                    <button
+                      onClick={() => {
+                        this.setState({
+                          showErrorDetails: !this.state.showErrorDetails,
+                        });
+                      }}
+                    >
+                      <OnlyIf test={this.state.showErrorDetails}>
+                        hide stack trace
+                      </OnlyIf>
+                      <OnlyIf test={!this.state.showErrorDetails}>
+                        show stack trace
+                      </OnlyIf>
+                    </button>
```

Comment:
6 lines for a conditional button text with usage of a custom `<OnlyIf>` component is a bit overkill. You could easily do something like:

```js
<button>
  {this.state.showErrorDetails ? "hide stack trace" : "show stack trace"}
</button>
```